Spring Quickstart: Add instructions for maven install steps

### Description

As a developer who is experienced to Java but new to Maven, I was stumped by this error for some time before I realized I needed to maven install the parent pom and test helper:

```
[ERROR] Failed to execute goal on project spring-rest-authz-resource-server: Could not resolve dependencies for project org.keycloak.quickstarts:spring-rest-authz-resource-server:jar:999.0.0-SNAPSHOT 
[ERROR] dependency: org.keycloak.quickstarts:keycloak-quickstarts-test-helper:jar:999.0.0-SNAPSHOT (test) 
[ERROR] Could not find artifact org.keycloak.quickstarts:keycloak-quickstarts-test-helper:jar:999.0.0-SNAPSHOT
```

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

--- a/spring/rest-authz-resource-server/README.md
+++ b/spring/rest-authz-resource-server/README.md
@@ -53,7 +53,15 @@ For more details, see the Keycloak documentation about how to [create a new real
 Build and Run the Quickstart
 -------------------------------
 
-If your server is up and running, perform the following steps to start the application:
+If this is the first time you're running the quickstart application, you will need to install the parent POM and test helper in your local Maven repository. From the root of the repository, do the following:
+
+```
+mvn install
+cd misc
+mvn install
+```
+
+If your Keycloak server is up and running, perform the following steps to start the application:
 
 1. Open a terminal and navigate to the root directory of this quickstart.
 

--- a/spring/rest-authz-resource-server/README.md
+++ b/spring/rest-authz-resource-server/README.md
@@ -58,7 +58,7 @@ If this is the first time you're running the quickstart application, you will ne
 ```
 mvn clean install
 cd misc
-mvn install
+mvn clean install
 ```
 
 If your Keycloak server is up and running, perform the following steps to start the application:

--- a/spring/rest-authz-resource-server/README.md
+++ b/spring/rest-authz-resource-server/README.md
@@ -56,7 +56,7 @@ Build and Run the Quickstart
 If this is the first time you're running the quickstart application, you will need to install the parent POM and test helper in your local Maven repository. From the root of the repository, do the following:
 
 ```
-mvn install
+mvn clean install
 cd misc
 mvn install
 ```

